code/src/style/checkbox.css

```diff
@@ -0,0 +1,4 @@
+.checkboxes {
+  margin-right: 20px;
+  margin-top: 20px;
+}
```

Comment:
Should we comment these code out for now 🤗

Reply:
Done